### ISSUE TITLE
Graph functions not updated to use GraphPoint objects

### ISSUE THREAD

**Issue by nwoods** [rootpy/rootpy#731]
A number of Graph functions still use syntax like `g[i][1]` instead of `g[i].y`. For example, my code threw an exception [here](https://github.com/rootpy/rootpy/blob/master/rootpy/plotting/graph.py#L352). 

The message was
```
  File "/afs/cern.ch/user/n/nawoods/ZZTools/recipe/vpython/lib/python2.7/site-packages/rootpy/plotting/graph.py", line 287, in __div__
    copy /= other
  File "/afs/cern.ch/user/n/nawoods/ZZTools/recipe/vpython/lib/python2.7/site-packages/rootpy/plotting/graph.py", line 352, in __idiv__
    (mypoint[1] / otherpoint[1]) *
TypeError: 'GraphPoint' object does not support indexing
```

**Comment by ndawe** [comment]
Sorry! I'll fix all that. Was a bit hasty in merging those changes in. Thanks for reporting!

**Comment by ndawe** [comment]
Fixed by #734
